horace_core/algorithms/set_instrument_horace.m

```diff
@@ -102,9 +105,13 @@
 end
 
 if nargout > 0
-    varargout = set_instr_or_sample_horace_(filename,'-instrument',instrument,argi{:});
+    sqw_list = set_instr_or_sample_horace_(filename,'-instrument',instrument,nargout,argi{:});
```

Comment:
I though we moving towards them. In anyway, this peace of code should be refactored by Chris. I do not want to created more complex conflicts then necessary

Reply:
You're right, sorry.

However, in this case, I would argue that this is not a flag, but a keyword argument. 

The decision also implied we should treat keyword args (i.e. an optional argument with value other than true/false) as keyword args rather than flags with an argument. I believe (although not stated explicitly) the intention is that flags only specify true/false.

Regardless, this is, as you say, for Chris to handle on refactoring.